src/app/GitUI/CommandsDialogs/CommitDialog/FormCommitTemplateSettings.Designer.cs

```diff
@@ -204,6 +210,29 @@ private void InitializeComponent()
         _NO_TRANSLATE_textBoxCommitTemplateName.TabIndex = 1;
         _NO_TRANSLATE_textBoxCommitTemplateName.TextChanged += textBoxCommitTemplateName_TextChanged;
         // 
+        // checkBoxRegexEnabled
+        // 
+        checkBoxRegexEnabled.Anchor = AnchorStyles.Bottom | AnchorStyles.Left;
+        checkBoxRegexEnabled.AutoSize = true;
+        checkBoxRegexEnabled.Location = new Point(3, 258);
+        checkBoxRegexEnabled.Margin = new Padding(3, 6, 3, 0);
+        checkBoxRegexEnabled.Name = "checkBoxRegexEnabled";
+        checkBoxRegexEnabled.RightToLeft = RightToLeft.Yes;
+        checkBoxRegexEnabled.Size = new Size(96, 19);
+        checkBoxRegexEnabled.TabIndex = 8;
+        checkBoxRegexEnabled.Text = "Regex enable";
+        toolTipRegex.SetToolTip(checkBoxRegexEnabled,
+            """
+                Use [[regex]](regex group number) to extract branch name part
+                Group number optional, defaults to 1
```

Comment:
the help text should have an example with group number, or only use first (non capturing groups can be done for other than the default)